OpenRA.Mods.Common/Traits/World/EditorResourceLayer.cs

```diff
@@ -113,6 +115,17 @@ public void UpdateCell(CPos cell)
 				Dirty.Add(cell + d);
 		}
 
+		void CalculateNetWorth()
```

Comment:
All the information you need is available from inside `UpdateDirtyTile`, so use that instead of `UpdateCell`.